clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/description/KubernetesManifestAnnotater.java

```diff
@@ -62,7 +63,15 @@ private static void storeAnnotation(Map<String, String> annotations, String key,
 
   public static void annotateManifestWithRelationships(KubernetesManifest manifest, KubernetesManifestSpinnakerRelationships relationships) {
     Map<String, String> annotations = manifest.getAnnotations();
+    annotateManifestWithRelationships(annotations, relationships);
 
+    manifest.getSpecTemplateAnnotations().flatMap((a) -> {
```

Comment:
Don't you usually omit the parens if there's only one parameter here? Just wondering.

Reply:
Good point

Reply:
Actually why is this a `flatMap` and not an `ifPresent` or something?

Reply:
Because this lets you avoid using a control flow structure. The fewer `if` statements the better